### ISSUE TITLE
elastic/filebeat has not been pushed to the helm repo

### ISSUE THREAD

**Issue by dperetti** [elastic/helm-charts#126]
```
helm repo add elastic https://helm.elastic.co
helm repo update
helm install --name filebeat elastic/filebeat --version 7.0.0-alpha1

Error: failed to download "elastic/filebeat"
```

**Comment by Crazybus** [comment]
This chart is still a WIP but will be pushed to the helm repo during the next release. https://github.com/elastic/helm-charts/pull/121 is the current PR that is putting the finishing touches on it. If you wanted to already play around with it you could install it directly from that branch. 

In retrospect merging in the docs with those instructions is a bit confusing for anyone browsing the charts from GitHub. Sorry about that!

**Comment by dperetti** [comment]
Meanwhile, got it to work by installing the chart from master. However, I had to comment out the `livenessProbe` that was killing the containers after 30s...
Monitoring on port 5066 is an X-Pack only feature, isn't it ?

**Comment by Crazybus** [comment]
> Meanwhile, got it to work by installing the chart from master. However, I had to comment out the livenessProbe that was killing the containers after 30s...

Yup this was already fixed in the pull request/branch that I recommended you install from. This pull request has been merged so it is also fixed in master.

> Monitoring on port 5066 is an X-Pack only feature, isn't it ?

Nope, it is also available in the oss release. There is an oss example which is also part of the automated testing for this chart to confirm that this setting and other future changes remains compatible with all versions and license types.  https://github.com/elastic/helm-charts/tree/master/filebeat/examples/oss
